databook/_toc.yml

```diff
@@ -42,6 +45,7 @@ chapters:
     - file: physiology/ophys/learning/learning                    # Also a placeholder that needs actual content
     - file: physiology/ophys/inhibitory-balance/inhibitory-balance                    # Also a placeholder that needs actual content
   - file: physiology/ephys/ephys-background                    # Also a placeholder that needs actual content
```

Comment:
ditto. I need the word "background" out of this file name.

Reply:
changes made